lib/SimpleHttpClient/SslClientConnection.cpp

```diff
@@ -72,6 +74,7 @@
 using namespace arangodb;
 using namespace arangodb::basics;
 using namespace arangodb::httpclient;
+using namespace std::chrono;
```

Comment:
std::chrono is used in just 3 places in this file. Does it really justify a `using namespace` directive?